src/_tutorials/server/get-started.md

```diff
@@ -134,25 +134,18 @@ More information:
 
 The steps above used the Dart VM (`dart`) to run the app. The Dart VM is
 optimized for fast, incremental compilation to provide instant feedback
-during development. Now that your small app is done, it's time to AOT compile your
-Dart code to optimized native machine code.
+during development. Now that your small app is done,
+it's time to AOT compile your Dart code to optimized native machine code.
 
-Use the `dart2aot` tool to AOT compile the program to machine code:
+Use the `dart2native` tool to AOT compile the program to machine code:
 
 ```terminal
-$ dart2aot bin/main.dart bin/main.dart.aot
+$ dart2native bin/main.dart -o bin/my_app
```

Comment:
I find it odd to use the `-o` argument to do nothing but remove the extension. To make this a bit more meaningful, can we change the output location, e.g. `-o out/my_app` ?

Reply:
It removes the extension AND changes the name, which would otherwise be bin/main.aot, right? That said, I don't mind changing the output directory if that's a good practice. (Is it, @domesticmouse?)

Reply:
Leaving the generated executable in the bin directory along side the source.dart file makes sense to me. It’s the technique I’m using in my Docker images.